docs/backup.md

```diff
@@ -38,6 +38,20 @@ Now you can move that file to your backup location.
 
 To restore the backup, just copy the backup file to the data folder of your new installation and rename it to `db.sqlite3`. Then start the Docker container.
 
+#### Custom `user`
+
+The default instructions assumes that the process has permissions to write to `/etc/linkding` (`root` by default). However, that may not be the case in case the containers is being ran with a custom `user`. In those cases, you can export the backup to  `/etc/linkding/data` where the same user should have permissions to write to.
```

Comment:
```suggestion
The default instructions assumes that the process has permissions to write to `/etc/linkding` (`root`). However, that may not be the case in case the container is being ran with `user` set. In those cases, you can export the backup to  `/etc/linkding/data` where the same user has write permissions.
```